main.cpp

```diff
@@ -56,51 +56,66 @@ Purpose:  This project will show you the difference between member functions and
 
 #include <iostream>
 #include <string>
+
 struct T
 {
-    T(<#type name#> v, const char* <#variable name#>)   //1
+    T(int inValue, const char *inName) : value(inValue), name(inName) {}   //1
```

Comment:
put the `*` next to the type name, not the variable name.
it makes it more clear that it's a pointer.   This also aligns with the JUCE Coding Standard
- [ ] revise